mkosi/config.py

```diff
@@ -4479,6 +4489,7 @@ def summary(config: Config) -> str:
                     Shim Bootloader: {config.shim_bootloader}
               Unified Kernel Images: {config.unified_kernel_images}
         Unified Kernel Image Format: {config.unified_kernel_image_format}
+        Unified Kernel Image Addons: {line_join_list(config.pe_addons)}
```

Comment:
A bit too late at this, but I think this should mention "PE" somehow, too, e.g. "Unified Kernel Image PE Addons", since the option is named `--pe-addon`, that is what I would grep for in the output of `mkosi summary`.

Reply:
Woops, forgot to rename that. Probably should only be "PE Addons" since that is what they are referred to in the spec.